obelisk/python/obelisk_py/zoo/robot/hardware/leap_hand/leap_node.py

```diff
@@ -0,0 +1,67 @@
+import os
+import sys
+import math
+
+file_dir = os.path.dirname(__file__)
+sys.path.append(file_dir)
+
+import dxl_motor_helper as dxl
+from dynamixel_sdk import *
+
+from obelisk_py.core.robot import ObeliskRobot
+from obelisk_py.core.obelisk_typing import ObeliskControlMsg, ObeliskSensorMsg
+from obelisk_sensor_msgs.msg import JointEncoders
+
+
+
+class ObeliskLeapHand(ObeliskRobot):
+
+    N_MOTORS = 16
+
+    def __init__(self, node_name):
+        super().__init__(node_name)
+        self.register_obk_publisher(
+            "pub_sensor_setting",
+            key="pub_sensor",
+            msg_type=JointEncoders,
+        )
+        self.register_obk_timer(
+            "timer_sensor_setting",
+            self.get_state,
+            key="timer_sensor",
+        )
+        dxl.setup(self.N_MOTORS)
+        dxl.sync_PID(range(self.N_MOTORS))
+        for i in range(self.N_MOTORS):
+            dxl.enable_motor(i)
+        self.yhat = [0 for _ in range(self.N_MOTORS)]
```

Comment:
for now, i think it's ok to leave the setup stuff here. if we need to expand general hardware setup stuff to a separate node for other systems, we can always handle that in a later PR.